README.md

```diff
@@ -68,6 +68,9 @@ objects such that:
 </dd>
 </dl>
 
+It is possible to use the `@CalledMethodsPredicate` for unsound bug-finding, rather than
+verification, by using the `!` (NOT) operator.
```

Comment:
This text doesn't say what is unsound, nor how to use the checker for unsound bug-finding.
I think it would be clearer to say that "! x" means: "it isn't the case that x was definitely called" or "x was not called on every possible path" rather than "x is not called" or "x was not called on any path".
The checker is sound with respect to this semantics, right?
It might be helpful to say that there is not currently a way to express "x was not called".  That is more specific than the given text.